src/PresentationalComponents/ZeroState/zeroStateConstants.js

```diff
@@ -139,7 +139,7 @@ const COMPLIANCE_ZERO_STATE = {
 
 const INSIGHTS_ZERO_STATE = {
     header: {
-        description: messages.insightsDescription,
+        description: 'Single, consistent management solution across on-premise, hybrid cloud, and public cloud. Included with Red Hat Enterprise Linux subscription. Single, consistent management solution across on-premise, hybrid cloud, and public cloud.',
```

Comment:
Since we have a constant, I think we need to use that. But instead of `messages.insightsDescription` we need to use `messages.defaultMessage` from here https://github.com/RedHatInsights/insights-dashboard/blob/master/src/Messages.js#L456

Reply:
So, here's what I'll do; those message constants are set up for `intl` which we are doing a soft deprecation on. I'll recreate that constant in a different format.